### PR TITLE
Update to integration tests

### PR DESCRIPTION
**Pull request by ruchim**
The ones I've added are draft-2, but will eventually update to draft-3!

This PR is contingent on [cron job](https://travis-ci.org/broadinstitute/cromwell/builds/385209234) going green.

### CODE REVIEW

File: centaur/src/main/resources/reference.conf [hunk 1]
```diff
@@ -35,7 +35,7 @@ centaur {
   # The timeout of the Centaur send/receive + unmarshal pipeline
   sendReceiveTimeout: 10 seconds
   # The maximal length of a workflow, intended as a sanity check and not actually a test in and of itself
-  maxWorkflowLength: 3 hours
+  maxWorkflowLength: 4 hours
```
Comment (kshakir) [inline centaur/src/main/resources/reference.conf]: This doesn't change anything afaik and may give folks a false sense of security. Travis is still going to 🔫  your jobs after 3 hours. Maybe if/when we move to another CI we'll get more time though.

Comment (ruchim) [reply centaur/src/main/resources/reference.conf]: true-- will revert!

File: centaur/src/main/resources/integrationTestCases/green/arrays/arrays.wdl [hunk 2]
```diff
@@ -1,36 +1,294 @@
...
+          cluster_file = gender_cluster_file,
+          disk_size = disk_size,
+          preemptible_tries = preemptible_tries
+      }
+  }
+
+ if (size(AutoCall.gtc_file) > 0) {
+  call GtcToVcf {
+    input:
+      vcf_filename = sub(sub(AutoCall.gtc_file, "gs://.*/", ""), ".gtc$", "") + ".vcf.gz",
```
[the remark this answers is not in the document]
Comment (ruchim) [reply centaur/src/main/resources/integrationTestCases/green/arrays/arrays.wdl]: I totally agree --we can pass that feedback along to Green -- but I've just copied/pasted their existing workflows from prod!

File: src/bin/ci/resources/papi_v1_application.conf [hunk 3]
```diff
@@ -2,6 +2,8 @@ include required(classpath("application.conf"))
 include "build_application.inc.conf"
 include "papi_application.inc.conf"
 
+system.input-read-limits.lines = 512000
```
Comment (kshakir) [inline src/bin/ci/resources/papi_v1_application.conf]: Changes to this file only affect Pipelines API `v1` tests. `v2` is gonna have a bad time.

Update the config for both in one place by adding it to `papi_application.inc.conf` listed just above.

File: src/bin/ci/resources/papi_application.inc.conf.ctmpl [hunk 4]
```diff
@@ -5,7 +5,7 @@ backend {
 }
 
 engine {
-  filesystems {
+  file
```
Comment (Horneth) [inline src/bin/ci/resources/papi_application.inc.conf.ctmpl]: Hmm I'm pretty sure this should be `filesystems` no ?

Comment (ruchim) [reply src/bin/ci/resources/papi_application.inc.conf.ctmpl]: yup! typo :(

File: centaur/src/main/resources/integrationTestCases/green/arrays/arrays.inputs [hunk 5]
```diff
@@ -1,39 +1,33 @@
...
+
+  "Arrays.ref_fasta": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta",
+  "Arrays.ref_fasta_index": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai",
+  "Arrays.ref_dict": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict",
+  "Arrays.dbSNP_vcf": "gs://broad-references/hg19/v0/dbsnp_138.b37.vcf.gz",
+  "Arrays.dbSNP_vcf_index": "gs://broad-references/hg19/v0/dbsnp_138.b37.vcf.gz.tbi",
+  "Arrays.haplotype_database_file": "gs://broad-references-private/hg19/v0/Homo_sapiens_assembly19.haplotype_database.txt",
+  "Arrays.variant_rsids_file": "gs://broad-references-private/fingerprinting_variant_rsids_v1.list",
+  "Arrays.preemptible_tries": 3
+}
```
Comment (cjllanwarne) [inline centaur/src/main/resources/integrationTestCases/green/arrays/arrays.inputs]: newline marker here. Khalid pointed this out to me recently - you might want to do the opposite to what this post suggests: https://intellij-support.jetbrains.com/hc/en-us/community/posts/207071445-On-save-new-line-added-to-end-of-file-How-stop-this-

**Comment by cjllanwarne** [comment]
Presumably these are running the draft-2 ones because that's what they're actually running as in production right now? 

Is it a huge overhead/burden to also turn on the draft-3 versions?

**Comment by kshakir** [comment]
> Is it a huge overhead/burden to also turn on the draft-3 versions?

Whenever the originals get updated, these should (in theory) be kept in sync.

The point of the CRON tests is to run as close as possible what the real world workflows are running. As many of the originals run in FC, I believe they should be draft-2 for now. If one wanted to additionally clone draft-3/1.0 versions I think that would be fine.

ToL: A better version of the CRON tests would just point-to/reference the originals from the source with smaller inputs, instead of having clones in this git repo.

**Pull request merged**